pages/signin/index.tsx

```diff
@@ -1,13 +1,86 @@
 import { Button } from '../../components/buttons';
+import { Card, CardContent, Typography, Container } from '@material-ui/core';
 import { Layout } from '../../containers/layout';
+import InputField from '../../components/input-field/InputField';
+import { useState } from 'react';
+import { PageProps, UserInput } from '../../lib/types';
+import { withTranslation } from '../../i18n';
+import styles from './Signin.module.css';
+import { useAuth } from '../../hooks/useAuth';
+import { FirebaseUser } from '../../lib/types/auth';
+
+const SignInPage = ({ t }: PageProps): React.ReactElement => {
+  const [formValues, setFormValues] = useState<Partial<UserInput>>({
+    name: {
+      lastName: '',
+    },
+    gender: 'OTHER',
+    email: '',
+    password: '',
+    type: 'TALENT',
+  });
+  const auth = useAuth();
+  console.log(auth.user);
+
+  const handleLogout = (e: React.MouseEvent<HTMLButtonElement>): void => {
+    e.preventDefault();
+    auth.logout();
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    if (formValues.email && formValues.password) {
+      auth
+        .signin(formValues.email, formValues.password)
+        .then((user: FirebaseUser | void | null) => {
+          if (typeof user !== null) {
+            console.log('current user: ', auth.user);
+          } else {
+            console.log(
+              'wrong email or password, current user still is: ',
+              auth.user,
+            );
+          }
+        })
+        .catch((error: Error) => console.error(error));
+    }
+  };
 
-const SignInPage = (): React.ReactElement => {
   return (
     <Layout title="sign in">
+      <Card>
+        <CardContent>
+          <Container>
+            <Typography variant="h2">Sign In</Typography>
+            <form onSubmit={handleSubmit}>
+              <InputField
```

Comment:
I'm getting a warning about the InputFields that I vaguely recall from earlier implementations. Should be gone after rebase. But perhaps make a mental note to check.

Reply:
OK